### ISSUE TITLE
Suggestion: kvantum theme

### ISSUE THREAD

**Issue by benneti** [hlissner/dotfiles#14]
Hey, 

I looked at your dotfiles for inspiration on how to get qt to respect the dracula theme and found this
``` nix
# Try really hard to get QT to respect my GTK theme.
env.GTK_DATA_PREFIX = [ "${config.system.path}" ];
env.QT_QPA_PLATFORMTHEME = "gtk2";
qt5 = { style = "gtk2"; platformTheme = "gtk2"; };
```
as for me this worked not too well (i.e. some remaining white backgrounds and so on).
I took a closer look at the dracula-theme package and found that it includes a kvantum theme which is supported by qtgnomeplatform.
To use this I had to do the following:
(I did not use the qt5 = ... as there the style gets set and for some reason overrides my personal session variables)
Install `qgnomeplatform`, set `QT_QPA_PLATFORMTHEME = "gnome"` and `QT_STYLE_OVERRIDE="kvantum"` and configure kvantum to use dracula
``` nix
  xdg.configFile = {
    "kvantum.kvconfig" = {
      text = "theme=Dracula";
      target = "Kvantum/kvantum.kvconfig";
    };
    "Dracula-kvantum" = {
      recursive = true;
      source = "${pkgs.dracula-theme}/share/themes/Dracula/kde/kvantum/Dracula";
      target = "Kvantum/Dracula";
    };
  };
```

**Comment by totoroot** [comment]
@benneti Thanks for this! This finally made theming for Qt apps work.
I set up the environment variables like you suggested, but chose to create a separate module for kvantum. If someone wants to go this route, [here](https://github.com/totoroot/dotfiles/blob/main/modules/desktop/kvantum.nix) is the configuration for all Kvantum Dracula themes in my dotfiles.
I noticed that `Dracula-purple-solid` was actually the theme that fit the look of GTK apps best.
